src/debug/daccess/dacimpl.h

```diff
@@ -14,6 +14,11 @@
 #ifndef __DACIMPL_H__
 #define __DACIMPL_H__
 
+// This header include will need to be removed as part of GitHub#12170.
+// The only reason it's here now is that this header references the GC-private
+// structure "HandleTableMap".
+#include "../../gc/objecthandle.h"
```

Comment:
I will be submitting a follow-up PR for this in a day or two - it's a little too large and off-topic to include as a part of this PR.